src/AttachmentPreview/AttachmentPreview.less

```diff
@@ -0,0 +1,195 @@
+@import (reference) "../less/utilities";
+
+@HEADER_BAR_HEIGHT: @size_5xl;
+@DESKTOP_BUFFER_SIZE: @size_l;
+@MOBILE_BUFFER_SIZE: @size_xs;
+
+// TODO: can I add this to sizing.less?
+@size_ml: 1.25rem;
+
+.AttachmentPreview--Background {
+  .zIndex--5;
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  background: fade(@neutral_black, 60%);
+}
+
+.AttachmentPreview--HeaderBar {
+  .zIndex--6;
+  background-color: @neutral_black;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: @HEADER_BAR_HEIGHT;
+  .alignItems(center);
+  color: @neutral_white;
+}
+
+.AttachmentPreview--AttachmentName {
+  line-height: @size_l;
+  // TODO: why isn't this adding the ellipsis?
+  .text--truncate();
+}
+
+.AttachmentPreview--FileIcon {
+  max-height: @size_l;
+  max-width: @size_l;
+  padding: @size_ml @size_s @size_ml @size_l;
+  .padding--left--l();
+  filter: brightness(0) invert(1); /* makes the file icon white without needing a different <img> src attr */
```

Comment:
Fancy! Works on all browsers we want to support?

Reply:
Yes, except for IE

Reply:
This is complements of @cofauver ; j